andela/mrm_api
# #164266587 An admin should be able to add structure id to room model

#### Description
Currently, there is no connection between the structure table and the Room table. This Pr adds that connection by adding a `structure_id` to the Room Model.

#### Type of change
- New feature (non-breaking change which adds functionality)

#### How Has This Been Tested?
- Pull this branch `story/164266587-add-strucutre-id-to-room-model`
- Run the `createRoom` mutation
- You should be able to create a room with a structureId.

#### Checklist:





#### Pivotal Tracker
[#164266587](https://www.pivotaltracker.com/story/show/164266587)
